lighthouse-core/report/html/renderer/category-renderer.js

```diff
@@ -170,7 +170,7 @@ class CategoryRenderer {
   /**
    * @param {LH.ReportResult.Category} category
    * @param {Record<string, LH.Result.ReportGroup>} groupDefinitions
-   * @return {Element}
+   * @return {DocumentFragment}
```

Comment:
Can this just remain Element? I don't see benefit in making the return type more specific.

Reply:
> Can this just remain Element? I don't see benefit in making the return type more specific.

It would be a `Node`, not an `Element`, so it has to change either way